.circleci/config.yml

```diff
@@ -22,8 +22,8 @@ jobs:
           name: vcpkg
           command: if [ $CIRCLE_BRANCH != "release" ]; then external/vcpkg/bootstrap-vcpkg.sh; mkdir build; ./external/vcpkg/vcpkg --x-install-root=$PWD/build/vcpkg_installed --triplet=x64-linux --feature-flags="manifests" install; fi
       - run:
-          name: Build the root version of the project
-          command: mkdir ../root_version; cp -Rf .git * ../root_version/; cd ../root_version; git checkout v6; git submodule update --init --recursive; cd build; cmake ../; make -j4
+          name: Build the old version of the project
+          command: mkdir ../root_version; cp -Rf .git * ../root_version/; cd ../root_version; git checkout et-v6.2.10; git submodule update --init --recursive; cd build; cmake ../; make -j4
```

Comment:
v6 branch was too old and had the catch2/glibc incompat issue on the vcpkg git commit for catch2.  Bump this up to a more recent backwards compat release.